packages/wrangler/src/d1/migrations/apply.tsx

```diff
@@ -206,7 +206,9 @@ Your database may not be available to serve requests during the migration, conti
 				)
 			);
 
-			if (errorNotes.length > 0) return;
+			if (errorNotes.length > 0) {
+				throw Error("Migration failed");
+			}
```

Comment:
It might be more appropriate to set `process.exitCode` to 1 here, as the error's already thrown above 🙂

Reply:
Thank you for the suggestion, I've updated accordingly.